tests/bootstrap.php

```diff
@@ -20,7 +20,7 @@
 require $test_root . '/includes/functions.php';
 
 tests_add_filter( 'muplugins_loaded', function () {
-	require dirname( __DIR__ ) . '/plugin.php';
+	require_once dirname( __DIR__ ) . '/plugin.php';
```

Comment:
✨ unrelated but needed to make the tests work when the plugin is already required.